stories/cli/steps.fmf

```diff
@@ -49,6 +49,15 @@
         story: 'Provision a container'
         example: tmt run provision --how=container --image=fedora:latest
 
+    /connect:
+        story: 'Connect to a provisioned box'
+        description:
+            Do not provision a new system. Instead, use provided
+            authentication data to connect to a running machine.
+        example:
+            - tmt run provision --how=connect --guest=name-or-ip --user=login --password=secret
+            - tmt run provision --how=connect --guest=name-or-ip --key=private-key-path
```

Comment:
Same here, is it supposed to be called `private-key-path` or `key` then? :)

Reply:
Yes, it should be called as it is shown in the example: `key` is the name and detailed description contains `private-key-path` to make it more clear.